docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -93,20 +93,21 @@ Now `ls-groups` can be run like any other CLI command.
 az ls-groups
 ```
 
-## Create an alias command with arguments
+### Create an alias command with arguments
```

Comment:
This must not be an H3 level title. All titles in this document must be at the H2 level for scannability. This is not reference material.